internal/tools/pipeline/providers/secret/interface.go

```diff
@@ -17,10 +17,12 @@ package secret
 import (
 	"context"
 
+	orgpb "github.com/erda-project/erda-proto-go/core/org/pb"
 	"github.com/erda-project/erda/internal/tools/pipeline/spec"
 )
 
 type Interface interface {
 	FetchSecrets(ctx context.Context, p *spec.Pipeline) (secrets, cmsDiceFiles map[string]string, holdOnKeys, encryptSecretKeys []string, err error)
 	FetchPlatformSecrets(ctx context.Context, p *spec.Pipeline, ignoreKeys []string) (map[string]string, error)
+	GetOrg(orgID uint64) (*orgpb.Org, error)
```

Comment:
Why `GetOrg` api exist in `secret` interface?

Reply:
adjusted to `debug` level

Reply:
moved to cache